generator/osm2meta.cpp

```diff
@@ -353,6 +353,14 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_drive_through(std::strin
   return {};
 }
 
+std::string MetadataTagProcessorImpl::ValidateAndFormat_self_service(std::string v)
+{
+  strings::AsciiToLower(v);
+  if (v == "yes" || v == "only" || v == "partially" || v == "no")
```

Comment:
Is there any metadata that is saved as an integer/enum instead of a string? It may take less space and reduce the risk of forgetting to change java code after editing cpp part and vice versa.

Reply:
I don't think so, maybe the only one was for wifi and that's strings.

![image](https://github.com/user-attachments/assets/61db3745-13e3-44aa-86f3-b491a1d40229)